### ISSUE TITLE
Fix S5332 FP: Don't raise http warning for SOAP actions

### ISSUE THREAD

**Issue by Bobsson** [SonarSource/sonar-dotnet#9017]
### Description

Similar to #4124, the S5332 rule should ignore the [SoapDocumentMethodAttribute's Action property](https://learn.microsoft.com/en-us/dotnet/api/system.web.services.protocols.soapdocumentmethodattribute.action?view=netframework-4.8.1#system-web-services-protocols-soapdocumentmethodattribute-action)

### Repro steps

Example taken from the MSDN page: 
```
[SoapDocumentMethod(Action="http://www.contoso.com/GetUserName")]
public string GetUserName() {
     return User.Identity.Name;
}
```

### Related information

We're currently using SQ Enterprise version 9.9, but I don't see any indications that it's been addressed since then.

**Comment by sebastien-marichal** [comment]
Hello @Bobsson,

Thank you for reporting this issue.

I confirm this is an FP.

**Comment by pavel-mikula-sonarsource** [comment]
Internal ticket [NET-1596](https://sonarsource.atlassian.net/browse/NET-1596)

[NET-1596]: https://sonarsource.atlassian.net/browse/NET-1596?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
